packages/fluentui/styles/tsconfig.json

```diff
@@ -2,6 +2,8 @@
   "extends": "../../../scripts/typescript/tsconfig.fluentui",
   "compilerOptions": {
     "composite": true,
+    "strict": true,
+    "strictNullChecks": true,
```

Comment:
I cannot enable it globally, but as this package deals a lot with `undefined` values I enabled it to have some help for compiler itself.